src/commcare_cloud/ansible/roles/riakcs/configure/tasks/main.yml

```diff
@@ -46,6 +51,9 @@
     mode=0640
   notify: "restart {{ name }}"
 
+- set_fact:
+    advanced_conf: "{{ _backend_vars.bitcask.advanced_conf }}"
```

Comment:
This will hardcode the bitcask backend for all environments, right? I do not think we want that as many of our environments use a different backend.